interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -264,6 +265,19 @@
         });
     });
 
+    function oeFormatShortDate(value){
```

Comment:
this could get a little confusing to have a javascript function name the same as the php. Would instead use oeFormatShortDate_js as name to make it clear. Do you think it would make sense to store this in a separate library file?(ie. do you envision needing to use this function anywhere else?)